content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -2,4 +2,1184 @@
 title: Writing Your First Component Test
 ---
 
-CONTENT TBD
+<!--
+<Alert type="info">
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
+- How to start testing components in Cypress.
+- How to mount a component and test that it renders correctly.
+- How to query the DOM, stub functions, and write assertions.
+- What passing and failing tests look like.
+- How to use the Cypress UI to help debug test results.
+
+</Alert>
+
+-->
+
+In order to get the most out of this guide, we recommend that you follow along
+by cloning the
+[cypress-component-testing-example](https://github.com/cypress-io/cypress-component-testing-examples)
+repo and then following the instructions at the top of the `README` of either of
+the following folders to get started:
+
+- React version:
+  [first-component-react](https://github.com/cypress-io/cypress-component-testing-examples/tree/main/first-component-react)
+- Vue version:
+  [first-component-vue](https://github.com/cypress-io/cypress-component-testing-examples/tree/main/first-component-vue)
+
+(more explanation about cloning the repo or getting started with one of the
+examples?)
```

Comment:
This is fine to do, but I think we can assume that readers would understand they could do this due to the content right above it.